tests/plugins/inlinetoolbar/manual/kama.html

```diff
@@ -0,0 +1,47 @@
+<style>
+	body {
+		margin-left: 0px;
+		padding-left: 400px;
```

Comment:
I assume that it is a know issue as this problem is solved the same way in `balloonpanel` tests.

Reply:
That was not a proper way to handle it, now we can do it the proper way so we need to put a proper reference, so that when looking at the code in couple of months we know exactly what's the reason for this.

Reply:
In other words, please report the issue in `ckeditor/ckeditor-dev` and reference it next to these workarounds in tests.